roles/edpm_iscsid/tasks/run.yml

```diff
@@ -52,7 +68,7 @@
     - not manage_iscsid_stat.changed|bool
     - iscsi_restart_stat.stat.exists|bool
   ansible.builtin.systemd:
-    name: edpm_iscsid
+    name: iscsid@edpm-compute.service
```

Comment:
is this the service name or is it edpm-compute@iscsid like used at line 49?

Reply:
Yeah, my bad. Corrected.